File: AplicacionPeliculas/Form1.cs
using System;
using System.Data;
using Microsoft.VisualBasic.CompilerServices;

namespace AplicacionPeliculas
{

    public partial class Form1
    {
        private string query;
        public int idbusca;

        public Form1()
        {
            InitializeComponent();
        }
        private void bloqueotextos()
        {
            txtId.ReadOnly = true;
            txtno.ReadOnly = true;
            txtNombreOriginal.ReadOnly = true;
            txtGenero.ReadOnly = true;
            txtFechaLanzamiento.ReadOnly = true;
            txtIdioma.ReadOnly = true;
            txtIdiomaOriginal.ReadOnly = true;
            txtCompania.ReadOnly = true;
            txtDuracion.ReadOnly = true;
            txtDescripcion.ReadOnly = true;
            comboAdult.Enabled = false;
            txtpais.ReadOnly = true;

        }
        private void Form1_Load(object sender, EventArgs e)
        {
            bloqueotextos();
            idbusca = My.MyProject.Forms.Peliculas.idpeli;
            busqueda();
        }
        public void busqueda()
        {
            DataTable dt;
            string query = "select * from VWPELICULA2 where idPelicula = ' " + idbusca.ToString() + "'";
            dt = Conecction.SelectQuery(query);


            txtId.Text = Conversions.ToString(dt.Rows[0][0]);
            txtno.Text = Conversions.ToString(dt.Rows[0][1]);
            txtFechaLanzamiento.Text = Conversions.ToString(dt.Rows[0][2]);
            txtDuracion.Text = Conversions.ToString(Operators.ConcatenateObject(dt.Rows[0][3], " Min."));
            comboAdult.Text = Conversions.ToString(dt.Rows[0][4]);
            txtIdiomaOriginal.Text = Conversions.ToString(dt.Rows[0][5]);
            txtpais.Text = Conversions.ToString(dt.Rows[0][6]);
            txtNombreOriginal.Text = Conversions.ToString(dt.Rows[0][7]);
            txtDescripcion.Text = Conversions.ToString(dt.Rows[0][8]);
            txtGenero.Text = Conversions.ToString(dt.Rows[0][9]);
            txtCompania.Text = Conversions.ToString(dt.Rows[0][10]);
            txtIdioma.Text = Conversions.ToString(dt.Rows[0][11]);

        }

        public void limpiar()
        {

        }

        private void PictureBox1_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.Pelicula.Show();
        }

        private void PictureBox2_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.Pelicula.Show();
        }

        private void PictureBox3_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.Pelicula.txtId.Visible = true;
            My.MyProject.Forms.Pelicula.Label7.Visible = true;
            My.MyProject.Forms.Pelicula.txtId.Text = idbusca.ToString();
            My.MyProject.Forms.Pelicula.Show();
        }
    }
}

File: AplicacionPeliculas/Form1.vb
Imports System.Collections.ObjectModel
Imports System.Data.SqlClient

Public Class Form1
    Dim query As String
    Public idbusca As Integer
    Private Sub bloqueotextos()
        txtId.ReadOnly = True
        txtno.ReadOnly = True
        txtNombreOriginal.ReadOnly = True
        txtGenero.ReadOnly = True
        txtFechaLanzamiento.ReadOnly = True
        txtIdioma.ReadOnly = True
        txtIdiomaOriginal.ReadOnly = True
        txtCompania.ReadOnly = True
        txtDuracion.ReadOnly = True
        txtDescripcion.ReadOnly = True
        comboAdult.Enabled = False
        txtpais.ReadOnly = True

    End Sub
    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        bloqueotextos()
        idbusca = Peliculas.idpeli
        busqueda()
    End Sub
    Public Sub busqueda()
        Dim dt As DataTable
        Dim query As String = "select * from VWPELICULA2 where idPelicula = ' " & idbusca.ToString & "'"
        dt = Conecction.SelectQuery(query)


        txtId.Text = dt.Rows(0)(0)
        txtno.Text = dt.Rows(0)(1)
        txtFechaLanzamiento.Text = dt.Rows(0)(2)
        txtDuracion.Text = dt.Rows(0)(3) & " Min."
        comboAdult.Text = dt.Rows(0)(4)
        txtIdiomaOriginal.Text = dt.Rows(0)(5)
        txtpais.Text = dt.Rows(0)(6)
        txtNombreOriginal.Text = dt.Rows(0)(7)
        txtDescripcion.Text = dt.Rows(0)(8)
        txtGenero.Text = dt.Rows(0)(9)
        txtCompania.Text = dt.Rows(0)(10)
        txtIdioma.Text = dt.Rows(0)(11)

    End Sub

    Sub limpiar()

    End Sub

    Private Sub PictureBox1_Click(sender As Object, e As EventArgs) Handles PictureBox1.Click
        Pelicula.Show()
    End Sub

    Private Sub PictureBox2_Click(sender As Object, e As EventArgs) Handles PictureBox2.Click
        Pelicula.Show()
    End Sub

    Private Sub PictureBox3_Click(sender As Object, e As EventArgs) Handles PictureBox3.Click
        Pelicula.txtId.Visible = True
        Pelicula.Label7.Visible = True
        Pelicula.txtId.Text = idbusca
        Pelicula.Show()
    End Sub
End Class
